MAINTAINERS.md

```diff
@@ -2,7 +2,7 @@
 
 See [CONTRIBUTING.md](./CONTRIBUTING.md) for general contribution guidelines.
 See [GOVERNANCE.md](./GOVERNANCE.md) for governance guidelines and maintainer responsibilities.
-See [CODEOWNERS](./CODEOWNERS) for a detailed list of owners for the various source directories.
+See [CODEOWNERS](https://github.com/bpfman/bpfman/blob/main/CODEOWNERS) for a detailed list of owners for the various source directories.
```

Comment:
I meant to ask you about this one.

This used to work from the MAINTAINERS.md file, but it didn't work for the website.
Try: https://bpfman.io/main/governance/maintainers/CODEOWNERS.

Also see the errors below:
```
$ mkdocs build
INFO    -  Cleaning site directory
INFO    -  Building documentation to directory: /home/afredette/go/src/github.com/bpfman/bpfman/site
INFO    -  Doc file 'blog/posts/sled-integration.md' contains an unrecognized relative link '[https://bpfman.io/main/governance/meetings/]', it was
           left as is.
INFO    -  Doc file 'governance/MAINTAINERS.md' contains an unrecognized relative link './CODEOWNERS', it was left as is.
INFO    -  Doc file 'governance/REVIEWING.md' contains an unrecognized relative link './CODEOWNERS', it was left as is.
INFO    -  Documentation built in 1.10 seconds
```

We could fix the compile error, by putting a CODEOWNERS file in docs/governance similar to the others, but then, since there's no extension, mkdocs doesn't know what to do with it.  It creates a link, but it asks you if you want to download the file instead of showing it to you.

If we CODEOWNERS could be a .md file, I could make it work for both cases, but I don't think we can. So, I thought that the best option was to link to the file on github.

Do you have another solution?